Tests/BowEffectsLaws/AsyncLaws.swift

```diff
@@ -3,24 +3,78 @@ import SwiftCheck
 import Bow
 import BowEffects
 
-class AsyncLaws<F: Async & EquatableK> where F.E: Arbitrary {
+public class AsyncLaws<F: Async & EquatableK> where F.E: Arbitrary {
     
-    static func check() {
+    public static func check() {
         success()
         error()
+        continueOnJumpsThreads()
+        asyncConstructor()
+        continueOnComprehension()
+        asyncCanBeDerivedFromAsyncF()
```

Comment:
missing `bracketReleaseIscalledOnCompletedOrError`, is it possible to add it?

Reply:
Oh I see. That test depends on `Promise`, which is still not created. I will add the test once Promise is added.

Reply:
Great!